CHANGELOG.md

```diff
@@ -8,6 +8,13 @@ All notable changes to this project will be documented in this file.
 
 ### Fixed
 
+## 1.3.1
+### Added
+* New <environment> parameter to pass environment variables to Dev App Server ([#183](../../pulls/183))
```

Comment:
can you also include a reference to the common lib PR that pull in env-vars from the appengine-web.xml?